test/unit/mdc-textfield/foundation.test.js

```diff
@@ -303,16 +309,16 @@ test('on blur removes mdc-text-field--focused class', () => {
 });
 
 test('on blur removes mdc-text-field__label--float-above when no input value present', () => {
-  const {mockAdapter, blur} = setupBlurTest();
+  const {blur, label} = setupBlurTest();
```

Comment:
Might want to adjust the descriptions of these tests too like you did above? (See also my comment about the `inputIsEmpty` name though)